manage

```diff
@@ -19,43 +19,30 @@ usage () {
   build - Build the docker images for the project.
           You need to do this first.
 
-  start - Creates the application containers from the built images
-          and starts the services based on the docker-compose.yml file.
-          Examples:
-          $0 start
-          $0 start <ip_proxy_1>,<ip_proxy_2>,<ip_proxy_3>,<ip_proxy_4> &
+  up - Starts all containers and docker-compose logs.
+       Use ctrl-c to exit logging. Use "down" or "stop" to stop the run.
+        Examples:
+        $0 start
+        $0 start <ip_proxy_1>,<ip_proxy_2>,<ip_proxy_3>,<ip_proxy_4> &
+  start - Same as up
+
+  logs - Display the logs from the docker compose run (ctrl-c to exit).
 
   cli - Start client container
 
-  stop - Stops the services.  This is a non-destructive process.  The containers
+  down - Brings down the services and removes the volumes (storage).  The containers
```

Comment:
The `down` command is destructive.  It removes the containers and volumes.  Please update documentation.